src/Driver/Encodings.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\DBAL\Driver;
+
+final class Encodings
+{
+    /**
+     * Represents whether a value should be bound with ASCII encoding
+     *
+     * @see \PDO::PARAM_STR_CHAR
```

Comment:
The existing code has references to PDO only for backward compatibility. The new code doesn't have to be and it definitely doesn't have to borrow values from PDO.